frontend/src/common/HelpDialog.tsx

```diff
@@ -0,0 +1,37 @@
+import HelpOutlineIcon from '@mui/icons-material/HelpOutline'
+import { Button, Dialog, DialogContent, DialogTitle, Slide, Typography } from '@mui/material'
+import { TransitionProps } from '@mui/material/transitions'
+import React, { ReactNode } from 'react'
+
+const Transition = React.forwardRef(function Transition(
+  props: TransitionProps & {
+    children: React.ReactElement<any, any>
+  },
+  ref: React.Ref<unknown>,
+) {
+  return <Slide direction='up' ref={ref} {...props} />
+})
+
+type Props = {
+  infoLabel: string
+  dialogTitle: string
+  content: ReactNode
+  icon?: ReactNode
+}
+
+function HelpDialog({ infoLabel, dialogTitle, content, icon }: Props) {
+  const [open, setOpen] = React.useState(false)
+
+  return (
+    <>
+      <Button size='small' onClick={() => setOpen(true)} endIcon={icon ? icon : <HelpOutlineIcon />}>
+        <Typography variant='caption'>{infoLabel}</Typography>
+      </Button>
+      <Dialog open={open} onClose={() => setOpen(false)} maxWidth='md' TransitionComponent={Transition}>
```

Comment:
All of our other dialogs prevent closing via the escape key or from clicking the background. Instead we include a close button at the bottom. Consider making this dialog work the same for consistency.

Reply:
I can see in some places we disable closing via the escape key but do we really disable by clicking outside? I don't think that's the case. There doesn't _seem_ to be much consistency in this. I can disable closing by the escape key for now, though.

Reply:
Never mind, we're not consistent at all with the onClose behaviour. Happy with you leaving as is if you prefer.